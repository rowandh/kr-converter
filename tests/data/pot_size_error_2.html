
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-3-29534465</p></td>
                                <td height="26" ><p align="center">&nbsp;2025-02-05 오후 9:48:47</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>abkhgxmtc8 (A 풀 하우스)</b></p></td>
<!--								<td ><p align="center">&nbsp;113640</p></td>   -->
								<td ><p align="center">&nbsp;113,640</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>HYEMIll</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>8 <font color='red'>◆</font>5 ♣6 <font color='red'>♥</font>A ♠6 ♣A <font color='red'>◆</font>9 [A 6 투 페어]  (다이) 
<br>* 시작 : [StageNo:29534465] [Credit:571,865원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:200]
<br>* NICKNAME:[hyemill]
<br>* 앤티: -5,000원(566,865원)
<br>* 홀 카드딜: <font color='red'>◆</font>8(20) <font color='red'>◆</font>5(17) [8 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:8 탑(<font color='red'>◆</font>8 <font color='red'>◆</font>5)]
<br>* 베팅: 콜 -5,000원(561,865원) - 베팅순서: [0][6]
[1085ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>8<font color='red'>◆</font>5) C (♣6<font color='red'>♥</font>A♠6)
<br>* 턴 시작: [플랍(1)] [족보:6 원 페어(♣6 ♠6 <font color='red'>♥</font>A <font color='red'>◆</font>8 <font color='red'>◆</font>5)]
<br>* 베팅: 체크 -0원(561,865원) - 베팅순서: [1][5]
[946ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>8<font color='red'>◆</font>5) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 턴 시작: [턴(2)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>8)]
<br>* 베팅: 체크 -0원(561,865원) - 베팅순서: [2][5]
[825ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>8<font color='red'>◆</font>5) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 턴 시작: [리버(3)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>9)]
<br>* 베팅: 다이 [0](561,865원) - 베팅순서: [3][5]
[295ms]
<br>* 종료: WinMoney[0원] Credit[561,865원]
<br>* 결과: 패배 [족보:A 6 투 페어] [카드:<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>9] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">561,865</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>juhyg</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>Q ♠9 ♣6 <font color='red'>♥</font>A ♠6 ♣A <font color='red'>◆</font>9 [A 9 투 페어]  
<br>* 시작 : [StageNo:29534465] [Credit:2,555,114원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[juhyg]
<br>* 앤티: -5,000원(2,550,114원)
<br>* 홀 카드딜: <font color='red'>♥</font>Q(37) ♠9(8) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(<font color='red'>♥</font>Q ♠9)]
<br>* 베팅: 콜 -5,000원(2,545,114원) - 베팅순서: [0][7]
[1252ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q♠9) C (♣6<font color='red'>♥</font>A♠6)
<br>* 턴 시작: [플랍(1)] [족보:6 원 페어(♣6 ♠6 <font color='red'>♥</font>A <font color='red'>♥</font>Q ♠9)]
<br>* 베팅: 체크 -0원(2,545,114원) - 베팅순서: [1][6]
[1938ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q♠9) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 턴 시작: [턴(2)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>♥</font>Q)]
<br>* 베팅: 체크 -0원(2,545,114원) - 베팅순서: [2][6]
[2396ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q♠9) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 턴 시작: [리버(3)] [족보:A 9 투 페어(<font color='red'>♥</font>A ♣A <font color='red'>◆</font>9 ♠9 <font color='red'>♥</font>Q)]
<br>* 베팅: 콜 -20,000원(2,525,114원) - 베팅순서: [3][6]
[1352ms]
<br>* 종료: WinMoney[0원] Credit[2,525,114원]
<br>* 결과: 패배 [족보:A 9 투 페어] [카드:<font color='red'>♥</font>A ♣A <font color='red'>◆</font>9 ♠9 <font color='red'>♥</font>Q]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-30,000</font></p></td>
											<td style="word-break:break-all"><p align="right">2,525,114</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>jsdohisud</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣K ♣7 [K 탑]  (다이) (Big Blind)
<br>* 시작 : [StageNo:29534465] [Credit:432,798원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[jsdohisud]
<br>* 앤티: -5,000원(427,798원)
<br>* 홀 카드딜: ♣K(51) ♣7(45) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♣K ♣7)]
<br>* 베팅: [블라인드:BIG] [금액:5,000원] [Creadit:422,798원]
<br>* 접속종료:[1][0]
<br>* 방나감(접속종료)

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">422,798</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>id382906aa</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠8 ♣3 ♣6 <font color='red'>♥</font>A ♠6 ♣A <font color='red'>◆</font>9 [A 6 투 페어]  (다이) (Small Blind)
<br>* 시작 : [StageNo:29534465] [Credit:417,952원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[id382906aa]
<br>* 앤티: -5,000원(412,952원)
<br>* 홀 카드딜: ♠8(7) ♣3(41) [8 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:8 탑(♠8 ♣3)]
<br>* 베팅: [블라인드:SMALL] [금액:5,000원] [Creadit:407,952원]
<br>* 베팅: 체크 -0원(407,952원) - 베팅순서: [0][8]
[771ms]
<br>* 커뮤니티 카드 딜: H(♠8♣3) C (♣6<font color='red'>♥</font>A♠6)
<br>* 턴 시작: [플랍(1)] [족보:6 원 페어(♣6 ♠6 <font color='red'>♥</font>A ♠8 ♣3)]
<br>* 베팅: 체크 -0원(407,952원) - 베팅순서: [1][1]
[2162ms]
<br>* 커뮤니티 카드 딜: H(♠8♣3) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 턴 시작: [턴(2)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 ♠8)]
<br>* 베팅: 체크 -0원(407,952원) - 베팅순서: [2][1]
[3051ms]
<br>* 커뮤니티 카드 딜: H(♠8♣3) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 턴 시작: [리버(3)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>9)]
<br>* 베팅: 체크 -0원(407,952원) - 베팅순서: [3][1]
[3935ms]
<br>* 베팅: 다이 [0](407,952원) - 베팅순서: [3][7]
[1433ms]
<br>* 종료: WinMoney[0원] Credit[407,952원]
<br>* 결과: 패배 [족보:A 6 투 페어] [카드:<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>9] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">407,952</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>mfgipsdyes</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠Q ♣J ♣6 <font color='red'>♥</font>A ♠6 ♣A <font color='red'>◆</font>9 [A 6 투 페어]  (다이) 
<br>* 시작 : [StageNo:29534465] [Credit:781,820원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:150]
<br>* NICKNAME:[mfgipsdyes]
<br>* 앤티: -5,000원(776,820원)
<br>* 홀 카드딜: ♠Q(11) ♣J(49) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(♠Q ♣J)]
<br>* 베팅: 콜 -5,000원(771,820원) - 베팅순서: [0][3]
[1609ms]
<br>* 커뮤니티 카드 딜: H(♠Q♣J) C (♣6<font color='red'>♥</font>A♠6)
<br>* 턴 시작: [플랍(1)] [족보:6 원 페어(♣6 ♠6 <font color='red'>♥</font>A ♠Q ♣J)]
<br>* 베팅: 체크 -0원(771,820원) - 베팅순서: [1][3]
[4914ms]
<br>* 커뮤니티 카드 딜: H(♠Q♣J) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 턴 시작: [턴(2)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 ♠Q)]
<br>* 베팅: 체크 -0원(771,820원) - 베팅순서: [2][3]
[6770ms]
<br>* 커뮤니티 카드 딜: H(♠Q♣J) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 턴 시작: [리버(3)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 ♠Q)]
<br>* 베팅: 다이 [1](771,820원) - 베팅순서: [3][3]
[7219ms]
<br>* 종료: WinMoney[0원] Credit[771,820원]
<br>* 결과: 패배 [족보:A 6 투 페어] [카드:<font color='red'>♥</font>A ♣A ♣6 ♠6 ♠Q] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">771,820</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>abkhgxmtc8</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠A ♠5 ♣6 <font color='red'>♥</font>A ♠6 ♣A <font color='red'>◆</font>9 [A 풀 하우스]  
<br>* 시작 : [StageNo:29534465] [Credit:1,536,290원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[abkhgxmtc8]
<br>* 앤티: -5,000원(1,531,290원)
<br>* 홀 카드딜: ♠A(0) ♠5(4) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(♠A ♠5)]
<br>* 베팅: 콜 -5,000원(1,526,290원) - 베팅순서: [0][1]
[2584ms]
<br>* 커뮤니티 카드 딜: H(♠A♠5) C (♣6<font color='red'>♥</font>A♠6)
<br>* 턴 시작: [플랍(1)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♠A ♣6 ♠6 ♠5)]
<br>* 베팅: 체크 -0원(1,526,290원) - 베팅순서: [1][2]
[2181ms]
<br>* 커뮤니티 카드 딜: H(♠A♠5) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 턴 시작: [턴(2)] [족보:A 풀 하우스(<font color='red'>♥</font>A ♣A ♠A ♣6 ♠6)]
<br>* 베팅: 체크 -0원(1,526,290원) - 베팅순서: [2][2]
[1760ms]
<br>* 커뮤니티 카드 딜: H(♠A♠5) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 턴 시작: [리버(3)] [족보:A 풀 하우스(<font color='red'>♥</font>A ♣A ♠A ♣6 ♠6)]
<br>* 베팅: [쿼터] (20,000원) Credit(1,506,290원) - 베팅순서: [3][2]
[3637ms]
<br>* 종료: WinMoney[113,640원] Credit[1,619,930원]
<br>* 결과: 승리 [족보:A 풀 하우스] [카드:<font color='red'>♥</font>A ♣A ♠A ♣6 ♠6]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>83,640</font></p></td>
											<td style="word-break:break-all"><p align="right">1,619,930</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>sdfmsjkdms4</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠K ♣2 [K 탑]  (다이) 
<br>* 시작 : [StageNo:29534465] [Credit:1,159,420원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:250]
<br>* NICKNAME:[sdfmsjkdms4]
<br>* 앤티: -5,000원(1,154,420원)
<br>* 홀 카드딜: ♠K(12) ♣2(40) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♠K ♣2)]
<br>* 베팅: 다이 [0](1,154,420원) - 베팅순서: [0][2]
[178ms]
<br>* 커뮤니티 카드 딜: H(♠K♣2) C (♣6<font color='red'>♥</font>A♠6)
<br>* 커뮤니티 카드 딜: H(♠K♣2) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 커뮤니티 카드 딜: H(♠K♣2) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 종료: WinMoney[0원] Credit[1,154,420원]
<br>* 결과: 패배 [족보:K 탑] [카드:♠K ♣2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">1,154,420</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Koijhhbvhbfghhh</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>J <font color='red'>◆</font>2 ♣6 <font color='red'>♥</font>A ♠6 ♣A <font color='red'>◆</font>9 [A 6 투 페어]  (다이) (Entry Fee)
<br>* 시작 : [StageNo:29534465] [Credit:500,000원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[koijhhbvhbfghhh]
<br>* 앤티: -5,000원(495,000원)
<br>* 홀 카드딜: <font color='red'>◆</font>J(23) <font color='red'>◆</font>2(14) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>◆</font>J <font color='red'>◆</font>2)]
<br>* 신규 참가비 [금액:5,000원] [Credit:490,000원]
<br>* 베팅: 체크 -0원(490,000원) - 베팅순서: [0][5]
[3932ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J<font color='red'>◆</font>2) C (♣6<font color='red'>♥</font>A♠6)
<br>* 턴 시작: [플랍(1)] [족보:6 원 페어(♣6 ♠6 <font color='red'>♥</font>A <font color='red'>◆</font>J <font color='red'>◆</font>2)]
<br>* 베팅: 체크 -0원(490,000원) - 베팅순서: [1][4]
[1659ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J<font color='red'>◆</font>2) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 턴 시작: [턴(2)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>J)]
<br>* 베팅: 체크 -0원(490,000원) - 베팅순서: [2][4]
[2298ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J<font color='red'>◆</font>2) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 턴 시작: [리버(3)] [족보:A 6 투 페어(<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>J)]
<br>* 베팅: 다이 [0](490,000원) - 베팅순서: [3][4]
[3077ms]
<br>* 종료: WinMoney[0원] Credit[490,000원]
<br>* 결과: 패배 [족보:A 6 투 페어] [카드:<font color='red'>♥</font>A ♣A ♣6 ♠6 <font color='red'>◆</font>J] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">490,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>dfgrdfgdc</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>J ♠3 [J 탑]  (다이) 
<br>* 시작 : [StageNo:29534465] [Credit:647,180원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[dfgrdfgdc]
<br>* 앤티: -5,000원(642,180원)
<br>* 홀 카드딜: <font color='red'>♥</font>J(36) ♠3(2) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>♥</font>J ♠3)]
<br>* 베팅: 다이 [0](642,180원) - 베팅순서: [0][4]
[746ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J♠3) C (♣6<font color='red'>♥</font>A♠6)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J♠3) C (♣6<font color='red'>♥</font>A♠6) (♣A)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J♠3) C (♣6<font color='red'>♥</font>A♠6) (♣A) (<font color='red'>◆</font>9)
<br>* 종료: WinMoney[0원] Credit[642,180원]
<br>* 결과: 패배 [족보:J 탑] [카드:<font color='red'>♥</font>J ♠3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">642,180</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-3-29534465">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


