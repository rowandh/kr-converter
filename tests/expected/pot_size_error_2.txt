PokerStars Hand #15329534465:  Hold'em No Limit ($5000/$5000) - 2025/02/05 21:48:47 KST
Table 'Table 1' 9-max Seat #6 is the button
Seat 1: abkhgxmtc8 ($1536290 in chips)
Seat 2: sdfmsjkdms4 ($1159420 in chips)
Seat 3: mfgipsdyes ($781820 in chips)
Seat 4: dfgrdfgdc ($647180 in chips)
Seat 5: Koijhhbvhbfghhh ($500000 in chips)
Seat 6: HYEMIll ($571865 in chips)
Seat 7: juhyg ($2555114 in chips)
Seat 8: id382906aa ($417952 in chips)
abkhgxmtc8: posts the ante $5000
sdfmsjkdms4: posts the ante $5000
mfgipsdyes: posts the ante $5000
dfgrdfgdc: posts the ante $5000
Koijhhbvhbfghhh: posts the ante $5000
HYEMIll: posts the ante $5000
juhyg: posts the ante $5000
id382906aa: posts the ante $5000
id382906aa: posts small blind $5000
Koijhhbvhbfghhh: posts big blind $5000
*** HOLE CARDS ***
abkhgxmtc8: calls $5000
sdfmsjkdms4: folds
mfgipsdyes: calls $5000
dfgrdfgdc: folds
Koijhhbvhbfghhh: checks
HYEMIll: calls $5000
juhyg: calls $5000
id382906aa: checks
*** FLOP *** [6c Ah 6s]
id382906aa: checks
abkhgxmtc8: checks
mfgipsdyes: checks
Koijhhbvhbfghhh: checks
HYEMIll: checks
juhyg: checks
*** TURN *** [6c Ah 6s] [Ac]
id382906aa: checks
abkhgxmtc8: checks
mfgipsdyes: checks
Koijhhbvhbfghhh: checks
HYEMIll: checks
juhyg: checks
*** RIVER *** [6c Ah 6s] [Ac] [9d]
id382906aa: checks
abkhgxmtc8: bets $20000
mfgipsdyes: folds
Koijhhbvhbfghhh: folds
HYEMIll: folds
juhyg: calls $20000
id382906aa: folds
abkhgxmtc8 collected $113640 from pot
*** SUMMARY ***
Total pot $120000 | Rake $6360
Board [6c Ah 6s Ac 9d]
Seat 1: abkhgxmtc8 showed [As 5s] and won ($113640)
Seat 2: sdfmsjkdms4 mucked [Ks 2c]
Seat 3: mfgipsdyes mucked [Qs Jc]
Seat 4: dfgrdfgdc mucked [Jh 3s]
Seat 5: Koijhhbvhbfghhh mucked [Jd 2d]
Seat 6: HYEMIll mucked [8d 5d]
Seat 7: juhyg showed [Qh 9s] and lost
Seat 8: id382906aa mucked [8s 3c]